docs/reference/argocd.md

```diff
@@ -957,10 +957,133 @@ spec:
 
 ## Resource Customizations
 
-The configuration to customize resource behavior. This property maps directly to the `resource.customizations` field in the `argocd-cm` ConfigMap.
+There are two ways to customize resource behavior: with subkeys (`resourceHealthChecks`, `resourceIgnoreDifferences`, and `resourceActions`) and without subkeys (`resourceCustomizations`). `resourceCustomizations` maps directly to the `resource.customizations` field in the `argocd-cm` ConfigMap, while each of the subkeys maps directly to their own field in the `argocd-cm`. `resourceHealthChecks` will map to `resource.customizations.health`, `resourceIgnoreDifferences` to `resource.customizations.ignoreDifferences`, and `resourceActions` to `resource.customizations.actions`
```

Comment:
The documentation is versioned, right? The user will need to know in which version the subkey config is available.